ubuntu-ruby-fips/README.md

```diff
@@ -17,7 +17,7 @@ This container image includes Ubuntu version `20.04` which contains the followin
 The Ubuntu container image will be built by a three-stage process: 
 
 1. The first stage builds the OpenSSL compiled with the FIPS 140-2 compliant OpenSSL module.
-1. The second stage builds the Ruby and Postgres client packages.
+1. The second stage builds the Ruby, and Postgres client packages.
```

Comment:
no need for the comma here because there are only two items in the list :)

Reply:
Done

Reply:
Still seeing the comma... you can remove the comma here.

Reply:
Hi @shulifink  the last commit https://github.com/cyberark/conjur-base-image/pull/15/commits/2378f20d054e22161d9045aa49c506e8413ee153 remove the comma, this file is outdated.